test/uk/gov/hmrc/incometaxsubscriptionselfemployedfrontend/controllers/BusinessListCYAControllerSpec.scala

```diff
@@ -107,6 +109,106 @@ class BusinessListCYAControllerSpec extends ControllerBaseSpec with MockIncomeTa
 
   }
 
+  "submit" should {
+    "return 303, SEE_OTHER" when {
+      "the connector returns successful json and submit with Yes option redirect to Initialse controller" in {
+        mockAuthSuccess()
+        mockGetSelfEmployments[Seq[SelfEmploymentData]]("Businesses")(Right(Some(Seq(
+          businessData
+        ))))
+
+        val result: Future[Result] = TestBusinessListCYAController.submit()(FakeRequest()
+          .withFormUrlEncodedBody(AddAnotherBusinessForm.addAnotherBusiness -> YesNoMapping.option_yes))
+
+        status(result) mustBe SEE_OTHER
+        redirectLocation(result) mustBe Some(routes.InitialiseController.initialise().url)
+
+      }
+
+      "the connector returns successful json and submit with No option redirect to Initialse controller" in {
```

Comment:
Could you rename this test to correctly match the scenario

Reply:
Done